### PR TITLE
[INLONG-627] [Doc] Translate http report guide (#627)

### LINKED ISSUE #627: [Feature][Doc] Translate http report guide
### Description

Translate HTTP report guide：

https://inlong.apache.org/docs/next/sdk/dataproxy-sdk/http

### Use case

_No response_

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by allendang001**
[[INLONG-627] [Doc] Translate http report guide]

Fixes #627

translate http report guide

### CODE REVIEW

**Comment by allendang001** [comment]
cc @dockerzhang

File: docs/sdk/dataproxy-sdk/http.md [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
-| 参数       | 含义       | 备注  |
-|----------|----------|-----|
-| groupId  | 数据流组 id  |     |
-| streamId | 数据流 ID   |     |
-| body     | 推送的数据内容  |     |
-| dt       | 推送的数据时间  |     |
-| cnt      | 推送条数     |     |
+| parameter | meaning                               | Remark  |
+|-----------|---------------------------------------|---------|
+| groupId   | data stream group id                  |         |
```
Comment (dockerzhang) [inline docs/sdk/dataproxy-sdk/http.md:17]: data->Data

File: docs/sdk/dataproxy-sdk/http.md [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
+| streamId  | data stream ID                        |         |
```
Comment (dockerzhang) [inline docs/sdk/dataproxy-sdk/http.md:18]: ditto

File: docs/sdk/dataproxy-sdk/http.md (hunk 1 above)
Comment (allendang001) [reply docs/sdk/dataproxy-sdk/http.md:17]: ok，l wiil change it

**Review by dockerzhang — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

**Pull request merged**